dts/bindings/rng/litex,prbs.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
would be good to follow the same header comment formatting as Kconfig here.
https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#header-comments-and-other-nits

```
# Copyright (c) 2019 Antmicro <www.antmicro.com>
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Formatting changed.